frontend/app/views/fragments/offer/option.scala.html

```diff
@@ -0,0 +1,10 @@
+@(title: String, offer: String, fullPrice: String, rightGutter: Boolean = false, note: String = "")
```

Comment:
:+1: Option is better here - we can then use a `map` rather an `if` condition below.